Datadog/E2ETests/Logging/LoggerBuilderE2ETests.swift

```diff
@@ -121,7 +204,14 @@ class LoggerBuilderE2ETests: E2ETests {
     }
 
     /// - api-surface: Logger.Builder.bundleWithTrace(_ enabled: Bool) -> Builder
-    func test_logs_logger_builder_BUNDLE_WITH_TRACE_enabled() { // E2E:wip
+    ///
+    /// - data monitor - unfortunately we can't assert any APM trait in this monitor, so we just check if the data comes in:
```

Comment:
it seems when enabled it adds `trace_id` and `span_id` to the log attributes, given there is active span. we cannot assert these attributes?

Reply:
Yes, when enabled we add two extra attributes to each `Log`:
```swift
static let traceID = "dd.trace_id"
static let spanID = "dd.span_id"
```
so it can create the link in Datadog. The `span` can be seen from Log UI, but these attributes are not preserved in the UI thus cannot be seen / asserted and the facet cannot be created for further querying. We have the same issue in Android monitors for `logs_logger_builder_bundle_with_trace_enabled` 🙁. @mariusc83 am I correct / do you see any other way around?